[Bug]: "DetachHead does not have the correct permissions to execute `UpdateRef`" error

### Is there an existing issue for this?

- [x] I have searched the existing issues

### What happened?

i get the following error when trying to use this action:

```
 Error: 
   0: DetachHead does not have the correct permissions to execute `UpdateRef`
   1: failed to sync upstream changes
Error: Process completed with exit code 1.
```

https://github.com/DetachHead/rebased/actions/runs/24606462489/job/71953876919

i followed [the instructions in the readme](https://github.com/vedantmgoyal9/winget-releaser#getting-started-)
and created a classic token with `public_repo`:

<img width="1160" height="558" alt="Image" src="https://github.com/user-attachments/assets/7a0e7b79-9681-46c7-8320-11dcae8bc33a" />

```yaml
name: Publish to package managers
on:
  release:
    types: [released]
jobs:
  winget:
    runs-on: ubuntu-slim
    steps:
      - uses: vedantmgoyal9/winget-releaser@7bd472be23763def6e16bd06cc8b1cdfab0e2fd5
        with:
          identifier: detachhead.rebased
          token: ${{ secrets.WINGET_TOKEN }}
          installers-regex: ^rebased.exe$
```

--- a/README.md
+++ b/README.md
@@ -23,7 +23,15 @@ expediting the amount of time it takes for a submission to be reviewed and publi
 > At least **one** version of your package should already be present in the [Windows Package Manager Community Repository][winget-pkgs-repo].
 > The action will use that version as a base to create manifests for new versions of the package.
 
-1. You will need to create a _classic_ Personal Access Token (PAT) with `public_repo` scope. _New_ fine-grained PATs aren't supported by the action. Review https://github.com/vedantmgoyal9/winget-releaser/issues/172 for information.
+1. You will need to create a _classic_ Personal Access Token (PAT) with the `public_repo` and `workflow` scopes. The `workflow` scope is required when synchronizing changes to workflow files from the upstream `microsoft/winget-pkgs` repository. Without it, the action may fail intermittently when synchronizing changes. The `workflow` permission can be omitted, but you will need to manually sync the fork when this happens. _New_ fine-grained PATs aren't supported by the action.
+
+   ```text
+   Error:
+      0: <username> does not have the correct permissions to execute `UpdateRef`
+      1: failed to sync upstream changes
+   ```
+
+   If you encounter this error, add the `workflow` scope to the classic PAT used by the `token` input or manually sync the `winget-pkgs` fork via the web UI, then rerun the workflow.
 
 2. Fork [microsoft/winget-pkgs][winget-pkgs-repo] under the same account/organization as the project's repository. If you are forking [winget-pkgs][winget-pkgs-repo] on a different account (e.g. bot/personal account), you can use the `fork-user` input to specify the username of the account where the fork is present.
 

